src/components/common/Profile.js

```diff
@@ -13,7 +13,7 @@ import {
 import { EllipsisOutlined } from '@ant-design/icons';
 import { useOktaAuth } from '@okta/okta-react/dist/OktaContext';
 import { useEffect } from 'react';
-import { useUserHook } from '../../hooks';
+import { useUserQuery } from '../../hooks';
```

Comment:
Glad to see this got updated! It was causing some issues when I tried to render the web app ealier